examples/io/plot_read_events.py

```diff
@@ -0,0 +1,49 @@
+"""
+.. _ex_read_events:
+
+=====================
+Reading an event file
+=====================
+
+Read events from a file. For a more detailed guide on how to read events
+using MNE-Python, see :ref:`tut_epoching_and_averaging`.
+"""
+# Authors: Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#          Chris Holdgraf <choldgraf@berkeley.edu>
+#
+# License: BSD (3-clause)
+
+import matplotlib.pyplot as plt
+import mne
+from mne.datasets import sample
+
+print(__doc__)
+
+data_path = sample.data_path()
+fname = data_path + '/MEG/sample/sample_audvis_raw-eve.fif'
+
+# Reading events
+# events = mne.read_events(fname)  # all
+fig, axs = plt.subplots(1, 3, figsize=(15, 5))
+
+events = mne.read_events(fname, include=1)
+mne.viz.plot_events(events, axes=axs[0], show=False)
+axs[0].set(title="restricted to event 1")
+
+events = mne.read_events(fname, include=[1, 2])
+mne.viz.plot_events(events, axes=axs[1], show=False)
+axs[1].set(title="restricted to event 1 or 2")
+
+events = mne.read_events(fname, exclude=[4, 32])
+mne.viz.plot_events(events, axes=axs[2], show=False)
+axs[2].set(title="keep all but 4 and 32")
+plt.setp([ax.get_xticklabels() for ax in axs], rotation=45)
+plt.tight_layout()
+plt.show()
+
+# Writing events
+mne.write_events('events.fif', events)
```

Comment:
proper names for event fif files should end with -eve.fif

some maybe use

example-eve.fif